pandas/io/pickle.py

```diff
@@ -72,6 +101,35 @@ def read_pickle(path, compression='infer'):
     Returns
     -------
     unpickled : type of object stored in file
+
+    Examples
+    --------
+    >>> original_df = pd.DataFrame({"foo": range(5), "bar": range(5, 10)})
+    >>> original_df
+       foo  bar
+    0    0    5
+    1    1    6
+    2    2    7
+    3    3    8
+    4    4    9
+    >>> pd.to_pickle(original_df, "./dummy.pkl")
+
+    >>> unpickled_df = pd.read_pickle("./dummy.pkl")
+    >>> unpickled_df
+       foo  bar
+    0    0    5
+    1    1    6
+    2    2    7
+    3    3    8
+    4    4    9
+
+    See Also
+    --------
+    pandas.DataFrame.to_pickle
+    pandas.Series.to_pickle
+    pandas.read_hdf,
```

Comment:
don't put commas at the end

Reply:
I think we add text describing each of these

Reply:
corrected.